CMakeLists.txt

```
@@ -14,6 +14,7 @@ option (BUILD_AGENT "Build EasyMesh agent" ON)
 option (BUILD_CONTROLLER "Build EasyMesh controller" ON)
 
 option(BUILD_SHARED_LIBS "Build shared libraries (.so) instead of static ones (.a)" ON)
+option (ENABLE_AMBIORIX "Include Bus agnostic API in the build" OFF)
```

As I mentioned orally:
```suggestion
option (ENABLE_HLAPI "Build the northbound (high-level) API" OFF)
```

Also please turn it on in at least one build in CI.

---

Applied. Will resolve as commit to alter CI to check build with flag is added.